index.ts

```diff
@@ -6,15 +6,15 @@ export * from './src/types.js';
  *
  * @since 0.0.6
  */
-export * as ID from './src/id.js';
+export * as Id from './src/id.js';
 
 /**
  * This module provides utility functions for working with base58 ids
  * in TypeScript.
  *
  * @since 0.0.6
  */
-export { BASE58_ALLOWED_CHARS, decodeBase58ToUUID, encodeBase58 } from './src/core/base58.js';
+export { decodeBase58ToUUID, encodeBase58 } from './src/core/base58.js';
```

Comment:
Does it make sense to Namespace these as part of this PR?

Reply:
agreed, just added a commit. In this case I think it's also fine to expose BASE58_ALLOWED_CHARS